storeshield/tests/utils.py

```
@@ -0,0 +1,73 @@
+    async def upload_file(
+        self,
+        file_path: str,
+        file_size: int,
+        content: StreamReader,
+        content_type: str = "application/octet-stream",
+    ) -> str:
+        """
+        Fake upload: reads the entire content from the stream and returns the file path.
+        """
+        # Read all content from the stream.
+        data = await content.read()
```

Ruff (0.8.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)